components/formats-gpl/src/loci/formats/in/LiFlimReader.java

```diff
@@ -91,9 +91,14 @@ public class LiFlimReader extends FormatReader {
 
   // relevant keys in timestamp table
   public static final String TIMESTAMP_KEY = "FLIMIMAGE: TIMESTAMPS - t";
+  
+  // relevant keys in fli 2.0
+  public static final String NUMBEROFFRAMES_KEY = "numberOfFrames";
+  public static final String NUMBEROFDARKIMAGES_KEY = "nrOfDarkImages";
+  public static final String PIXELFORMAT_KEY = "pixelFormat";
 
   // supported versions
-  public static final String[] KNOWN_VERSIONS = {"1.0"};
+  public static final String[] KNOWN_VERSIONS = {"1.0", "2.0"};
```

Comment:
In terms of documentation, I assume we could add a `versions = 1.0, 2.0` to https://github.com/ome/bio-formats-documentation/blob/9c1b017fb6d6907bb0c62b247f8d1ae50cc9fab5/src/main/resources/format-pages.txt#L1312-L1325 so that the format pages communicate the support expectations appropriately?